static/js/components/dashboard/CourseDescription.js

```diff
@@ -69,13 +67,16 @@ export default class CourseDescription extends React.Component {
     return _.compact([dateMessage, additionalDetail]);
   }
 
-  renderViewCourseLink = (courseRun: CourseRun): React$Element<*>|null => (
-    (courseRun && courseRun.course_id) ?
-      <a href={`${edxLinkBase}${courseRun.course_id}`} target="_blank">
+  renderViewCourseLink = (courseRun: CourseRun): React$Element<*>|null => {
+    const edxLinkBase = SETTINGS.edx_base_url;
+    const edxCourseLink = _.endsWith(edxLinkBase, '/') ? `${edxLinkBase}courses/` : `${edxLinkBase}/courses/`;
```

Comment:
Can you use the `url-join` library instead? It's already in `package.json` so you should just need to import it